packages/gatsby-plugin-sharp/src/duotone.js

```diff
@@ -108,9 +108,12 @@ async function overlayDuotone(
     .toBuffer()
 
   return await originalImage
-    .overlayWith(duotoneWithTransparency, {
-      raw: { width: info.width, height: info.height, channels: 4 },
-    })
+    .composite([
```

Comment:
I'm going through the commit where overlayWith is being removed https://github.com/lovell/sharp/commit/631a3597c7bbca3cd3f4d1c76febad50f6d8cf44#diff-131b66be84940637abfa0952789d422bL157-L165 .

Let's do the same trick has they do and force `blend: 'over'`. It's the default blend mode but just to make sure we don't fail in another update of sharp where it might have changed.

Reply:
that seems reasonable.  I'd highlight that the added snapshot tests would detect any change in rendering as well, so it would detect such a breaking change.

Reply:
oh yeah that's a good point! Anyway thanks for updating! 😄